staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -129,7 +129,22 @@ func (a *AdmissionOptions) ApplyTo(
 }
 
 func (a *AdmissionOptions) Validate() []error {
+	if a == nil {
+		return nil
+	}
+
 	errs := []error{}
+	registeredPluginsMap := make(map[string]bool)
+	registeredPlugins := a.Plugins.Registered()
+	for _, name := range registeredPlugins {
+		registeredPluginsMap[name] = true
+	}
+
+	for _, name := range a.PluginNames {
+		if registeredPluginsMap[name] != true {
```

Comment:
Better to use `if !registeredPluginsMap[name]`.

Reply:
ok